sys/net/network_layer/ng_ipv6/addr/ng_ipv6_addr.c

```diff
@@ -75,7 +73,7 @@ void ng_ipv6_addr_init_prefix(ng_ipv6_addr_t *out, const ng_ipv6_addr_t *prefix,
     if (bits % 8) {
         uint8_t mask = 0xff << (8 - (bits - (bytes * 8)));
 
-        out->u8[bytes] = prefix->u8[bytes] & mask;
+        out->u8[bytes] |= prefix->u8[bytes] & mask;
```

Comment:
This is a no op. :) Probably you meant `out->u8[bytes] &= mask` or `out->u8[bytes] &= ~mask`.